code/datums/components/seethrough_mob.dm

```diff
@@ -0,0 +1,124 @@
+///A component that lets you turn your character transparent in order to see and click through yourself.
+/datum/component/seethrough_mob
+	///The atom that enables our dark magic
+	var/obj/render_source_atom
```

Comment:
```suggestion
	var/atom/movable/render_source_atom
```
less variables, more lightweight (disregard in case of unknown variables).

Reply:
The game got upset with me when I tried sending a atom/movable to nullspace.

Reply:
What gives, any more detail on this?

Reply:
...However, testing it right now, it seemed fine this time. I probably tried atom instead of atom/movable or something.